src/components/description-data/index.js

```diff
@@ -0,0 +1,61 @@
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import '../grid';
+import styles from './styles.scss';
+import './components/description-data-item';
+
+export class VlDescriptionData extends LitElement {
+  static get styles() {
+    return [
+      css`
+        ${unsafeCSS(styles)}
+      `,
+    ];
+  }
+
+  static get properties() {
+    return {
+      size: { type: String, attribute: 'data-vl-items-size', reflect: true },
+      maxSize: { type: String, attribute: 'data-vl-items-max-size', reflect: true },
+      mediumSize: { type: String, attribute: 'data-vl-items-medium-size', reflect: true },
+      mediumMaxSize: { type: String, attribute: 'data-vl-items-medium-max-size', reflect: true },
+      smallSize: { type: String, attribute: 'data-vl-items-small-size', reflect: true },
+      smallMaxSize: { type: String, attribute: 'data-vl-items-small-max-size', reflect: true },
+      extraSmallSize: { type: String, attribute: 'data-vl-items-extra-small-size', reflect: true },
+      extraSmallMaxSize: { type: String, attribute: 'data-vl-items-extra-small-max-size', reflect: true },
+    };
+  }
+
+  firstUpdated() {
+    const observer = new MutationObserver(() => {
+      this.requestUpdate();
+    });
+
+    observer.observe(this, { subtree: true, childList: true });
+  }
+
+  render() {
+    return html`<div class="vl-description-data">
+      <div is="vl-grid">
+        ${[...this.children].map((child, index) => {
```

Comment:
Deze functionaliteit, die mapt over uw children en die via een slot uw child in uw `vl-column` wrapped, hebben we ook gebruikt in de `vl-breadcrumb`. Zouden we deze kunnen abstraheren en op 1 plaats zetten om te beheren? Onze utils mag van mij gerust een `index` file krijgen met util functies die we kunnen gebruiken in onze componenten!

Reply:
Is bij breadcrumb geen column, mss samen eens bekijken binnenkort

Reply:
Is goed!